<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../docs/img/racoonized_logo.png</normaloff>
    <normalon>../../docs/img/racoonized_logo2.png</normalon>../../docs/img/racoonized_logo.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>50</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Output Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>450</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_output">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>150</y>
      <width>450</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_seleccionar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar</string>
    </property>
    <property name="icon">
     <iconset theme="insert-image">
      <normaloff>../computer_vision/Analizador/Analizador_imagen/src/MainWindow</normaloff>../computer_vision/Analizador/Analizador_imagen/src/MainWindow</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_restaurar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Restaurar</string>
    </property>
    <property name="icon">
     <iconset theme="edit-undo">
      <normaloff>../computer_vision/Analizador/Analizador_imagen/src/MainWindow</normaloff>../computer_vision/Analizador/Analizador_imagen/src/MainWindow</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>-255</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelValue">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Control del Escalar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTransformaciones">
    <property name="title">
     <string>Transformaciones</string>
    </property>
<<<<<<< HEAD
    <widget class="QMenu" name="menuFiltros_Tonalidades">
     <property name="title">
      <string>Conversi贸n y Tono</string>
     </property>
     <addaction name="action_BN"/>
     <addaction name="action_Negativo"/>
     <addaction name="action_Binario"/>
     <addaction name="action_BinarioInv"/>
     <addaction name="action_Posterizado"/>
=======
    <widget class="QMenu" name="menuConversi_n">
     <property name="title">
      <string>Conversi贸n</string>
     </property>
     <addaction name="actionBN"/>
     <addaction name="actionNegativo"/>
     <addaction name="actionPosterizado"/>
    </widget>
    <widget class="QMenu" name="menuUmbralizado">
     <property name="title">
      <string>Umbralizado</string>
     </property>
     <addaction name="actionBinario"/>
     <addaction name="actionBinarioInv"/>
>>>>>>> temp-actualizaciones
    </widget>
    <widget class="QMenu" name="menuAjuste">
     <property name="title">
      <string>Ajuste</string>
     </property>
<<<<<<< HEAD
     <addaction name="action_Intensidad"/>
     <addaction name="action_Contraste"/>
=======
     <addaction name="actionBrillo"/>
     <addaction name="actionContraste"/>
>>>>>>> temp-actualizaciones
    </widget>
    <widget class="QMenu" name="menuEspeciales">
     <property name="title">
      <string>Especiales</string>
     </property>
<<<<<<< HEAD
     <addaction name="action_Parabola"/>
     <addaction name="action_ParabolaInv"/>
    </widget>
    <addaction name="menuFiltros_Tonalidades"/>
    <addaction name="menuAjuste"/>
    <addaction name="menuEspeciales"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos_ML">
    <property name="title">
     <string>Algoritmos </string>
=======
     <addaction name="actionParabola"/>
     <addaction name="actionParabolaInv"/>
    </widget>
    <addaction name="menuConversi_n"/>
    <addaction name="menuUmbralizado"/>
    <addaction name="menuAjuste"/>
    <addaction name="menuEspeciales"/>
   </widget>
   <widget class="QMenu" name="menuPosici_n">
    <property name="title">
     <string>Posici贸n</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos_ML">
    <property name="title">
     <string>Algoritmos</string>
>>>>>>> temp-actualizaciones
    </property>
   </widget>
   <widget class="QMenu" name="menuInformaci_n">
    <property name="title">
     <string>Informaci贸n</string>
    </property>
    <addaction name="mostrarInfo"/>
    <addaction name="ver_bitmap"/>
   </widget>
<<<<<<< HEAD
   <widget class="QMenu" name="menuTransformaciones_Geom_tricas">
=======
   <widget class="QMenu" name="menuOperaciones">
>>>>>>> temp-actualizaciones
    <property name="title">
     <string>Operaciones</string>
    </property>
    <addaction name="mostrarOperaciones"/>
   </widget>
   <addaction name="menuOpciones"/>
   <addaction name="menuTransformaciones"/>
<<<<<<< HEAD
   <addaction name="menuTransformaciones_Geom_tricas"/>
=======
   <addaction name="menuOperaciones"/>
   <addaction name="menuPosici_n"/>
>>>>>>> temp-actualizaciones
   <addaction name="menuAlgoritmos_ML"/>
   <addaction name="menuInformaci_n"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mostrarInfo">
   <property name="text">
    <string>Mostrar</string>
   </property>
  </action>
  <action name="ver_bitmap">
   <property name="text">
    <string>Ver Bitmap</string>
   </property>
  </action>
  <action name="actionBinario">
   <property name="text">
    <string>Binario</string>
   </property>
  </action>
  <action name="actionBN">
   <property name="text">
    <string>Blanco y Negro</string>
   </property>
  </action>
  <action name="actionBinarioInv">
   <property name="text">
    <string>Binario Invertido</string>
   </property>
  </action>
  <action name="actionNegativo">
   <property name="text">
    <string>Negativo</string>
   </property>
  </action>
  <action name="actionBrillo">
   <property name="text">
    <string>Brillo</string>
   </property>
  </action>
  <action name="actionContraste">
   <property name="text">
    <string>Contraste</string>
   </property>
  </action>
  <action name="actionParabola">
   <property name="text">
    <string>Parabola</string>
   </property>
  </action>
  <action name="actionParabolaInv">
   <property name="text">
    <string>Parabola Invertida</string>
   </property>
  </action>
  <action name="actionPosterizado">
   <property name="text">
    <string>Posterizado</string>
   </property>
  </action>
<<<<<<< HEAD
  <action name="action_BN">
   <property name="text">
    <string>Blanco y Negro</string>
   </property>
  </action>
  <action name="action_Negativo">
   <property name="text">
    <string>Negativo</string>
   </property>
  </action>
  <action name="action_Binario">
   <property name="text">
    <string>Binario</string>
   </property>
  </action>
  <action name="action_BinarioInv">
   <property name="text">
    <string>Binario Invertido</string>
   </property>
  </action>
  <action name="action_Posterizado">
   <property name="text">
    <string>Posterizado</string>
   </property>
  </action>
  <action name="action_Intensidad">
   <property name="text">
    <string>Ajustar Brillo</string>
   </property>
  </action>
  <action name="action_Contraste">
   <property name="text">
    <string>Ajustar Contraste</string>
   </property>
  </action>
  <action name="action_Parabola">
   <property name="text">
    <string>Par谩bola</string>
   </property>
  </action>
  <action name="action_ParabolaInv">
   <property name="text">
    <string>Par谩bola Invertida</string>
   </property>
  </action>
  <action name="mostrarOperaciones">
   <property name="text">
    <string>Abrir Men煤</string>
=======
  <action name="mostrarOperaciones">
   <property name="text">
    <string>Mostrar</string>
>>>>>>> temp-actualizaciones
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
